### PR TITLE
Add link to specific yq github repo

### PR DESCRIPTION
**Pull request by wheelerlaw**
There are two widely-used versions of `yq`, one written by [mikefarah](https://github.com/mikefarah/yq) and another written by [kislyuk](https://github.com/kislyuk/yq). This adds a link to the specific version used in this runner.

# Description
New tool, Bug fixing, or Improvement?  
Please include a summary of the change and which issue is fixed. Also include relevant motivation and context.  
**For new tools, please provide total size and installation time.**

## Check list
- [ ] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/linux/Ubuntu2004-Readme.md [hunk 1]
```diff
@@ -100,7 +100,7 @@
 - SVN 1.13.0
 - Terraform 1.1.4
 - yamllint 1.26.3
-- yq 4.16.2
```
Comment (GitesParti343) [inline images/linux/Ubuntu2004-Readme.md:103]: [yq]

Comment (GitesParti343) [reply images/linux/Ubuntu2004-Readme.md:103]: yq

File: images/linux/Ubuntu2004-Readme.md [hunk 2]
```diff
@@ -100,7 +100,7 @@
...
+- [yq](https://github.com/mikefarah/yq) 4.16.2
```
Comment (al-cheb) [inline images/linux/Ubuntu2004-Readme.md:103]: You have to update `https://github.com/actions/virtual-environments/blob/92eeb55189d6faee20e7e2df1eb741fb5c140572/images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1#L292`, this is autogenerated documentation file.

Comment (al-cheb) [reply images/linux/Ubuntu2004-Readme.md:103]: I think the default output is good:
```
$ ./yq_linux_amd64 --version
yq (https://github.com/mikefarah/yq/) version 4.17.2
```

Comment (al-cheb) [reply images/linux/Ubuntu2004-Readme.md:103]: @wheelerlaw, Could you please make the changes?

**Comment by miketimofeev** [comment]
@wheelerlaw feel free to reopen once the changes mentioned in the @al-cheb comment are applied

**Pull request closed without merging**